webapp/components/emoji_picker/emoji_picker.jsx

```diff
@@ -0,0 +1,449 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import $ from 'jquery';
```

Comment:
Like my above comment, we are trying to avoid using jQuery. Can you replace any uses of it if at all possible?